.gitignore

```diff
@@ -30,9 +30,10 @@ coverage
 # Dotenv
 **/*.env*
 !.env.sample
+!.env.development.sample
```

Comment:
Is there a reason to keep this in the .development .env rather than the root one, given that we also need this env var on Prod?  I know it's a manual process to add it there, I just mean for documentation purposes, seems unclear that it would be in development rather than all envs (or just production for that matter.  Thoughts?

Reply:
Good question. We don't want to enable Sentry on development, do we? That'll be quite a lot of noise. I was thinking of having the required variables for Production in `.env` and override them locally with `.env.development` (`.env.local` will work too). That's why the code checks if the environment is `production` and if `process.env.REACT_APP_SENTRY_DSN` is set. 

Does that make sense?

Reply:
Ah, yeah, perfect.  Good plan!

Reply:
Can you add a comment to the readme that both .env.sample and .env.development will need to be copied during setup?

Reply:
Done.